app/scripts/portland_ingest.py

```diff
@@ -0,0 +1,158 @@
+"""
+OBJECTIVE: Ingest Portland Ridership Data into our Database
+AUTHOR: Jimena Salinas
+DATE: 05/06/2024
+
+SOURCES: the data sets were obtained by a direct request to the 
+TriMet Public Records department. This is the link to the form
+to request data sets for Tri Met: 
+https://trimet.org/publicrecords/recordsrequest.htm
+The  Receipt ID for this Public Records Request is PRR 2024-254.
+
+VARIABLES:
+    - ID (int): this ID is the foreign key from the TransitStations table,
+    it can be obtained using city = 'PDX' and station_id
+    - station_id (from JSON raw file): (str) a stop_id (GTFS), only needed to 
+    get the foreign key, it does not go into the ridership db table
+    - date: (datetime) represents a day in the format yyyy-mm-dd, only includes
+    values for 2023
+    - ridership: (int) represents the average number of bus and light rail Tri Met
+    riders. The riders value was originally provided as an average by season 
+    and day type (Saturday, Sunday, and Weekday). In order to fit the data model
+    for the project which is at the datetime level, the data has been mapped to dates in 2023.
+"""
+
+import json
+import sys
+import os
+from requests.models import Response
+import django
+import datetime
+from datetime import datetime
+import pytz
+from typing import List, Dict, Any
+from django.db.utils import IntegrityError
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+parent_dir = os.path.abspath(os.path.join(current_dir, ".."))
+sys.path.append(parent_dir)
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "geodjango.settings")
+
+django.setup()
+
+
+from route_rangers_api.models import TransitStation, RidershipStation
+
+
+def get_transit_station_ids_pdx() -> list:
+    """
+    Get a list of all id values from the TransitStation table for PDX
+    """
+    return list(TransitStation.objects.filter(city="PDX").values_list("id", flat=True))
```

Comment:
A query set is already an iterable object so I don't think is necessary to convert to a list.